test/topology_custom/test_view_build_status.py

```diff
@@ -489,3 +489,63 @@ async def rows_migrated():
         return (len(result) == 3) or None
 
     await wait_for(rows_migrated, time.time() + 60)
+
+# Reproduces scylladb/scylladb#20754
```

Comment:
Please provide a short summary of what the issue was about, what the test does and how it reproduces the issue.

The important but not really obvious thing to those which do not know what issue 20754 is about is that the bug happens right after the last node is upgrade, i.e. migration to group 0 might run too quickly, producing an error. I'm not sure this can be easily inferred from the test code alone.